### PR TITLE
Update for v5

### PR DESCRIPTION
**Pull request by gurgunday**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,35 +30,35 @@
...
     "end-of-stream": "^1.4.4",
     "fast-content-type-parse": "^1.1.0",
-    "fast-querystring": "^1.0.0",
-    "fastify-plugin": "^4.0.0",
+    "fast-querystring": "^1.1.2",
+    "fastify-plugin": "^4.5.1",
     "pump": "^3.0.0",
     "toad-cache": "^3.7.0",
-    "undici": "^5.19.1"
+    "undici": "^5.28.3"
```
Comment (mcollina) [inline package.json:61]: Bump this to v6

**Review by mcollina — COMMENTED** [review]
Can you also update tap?

I think I'll likely ship this sooner to expose the updated undici.

File: package.json (hunk 1 above)
Comment (gurgunday) [reply package.json:61]: Will do both

**Comment by gurgunday** [comment]
There are multiple errors when tap is updated to v18, and it doesn't seem trivial to fix them...

If people really need the release, we can follow-up later to upgrade `devDeps` separately

**Comment by gurgunday** [comment]
Done

**Comment by elmarcusec** [comment]
@gurgunday @mcollina I have a question. When and in which version do you plan to release changes to upgrade the undici library?

**Comment by gurgunday** [comment]
This will be released as a major

**Comment by gurgunday** [comment]
Ok I made a mistake, should've merged to next first 😄

Will fix in a sec

**Comment by gurgunday** [comment]
This PR is ready, only note I'll leave is that for some reason, in certain cases `t.teardown` is not functioning and I have to resolve the promises myself, not sure if this is a tap issue or what

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**